app/views/items/_form.html.erb

```diff
@@ -0,0 +1,4 @@
+<%= simple_form_for [@user, @list, @list.items.build] do |f| %>
```

Comment:
Use shallower routes so there aren't so many dependencies needed for generating URLs.